lib/merchant.rb

```diff
@@ -22,4 +22,19 @@ def invoices
     @invoices ||= @repository.find_invoices(id)
   end
 
+  def revenue
+    successful_invoice_items =  successful_invoices.map { |i| i.invoice_items }.flatten
```

Comment:
Line is too long. [87/80]<br>Unnecessary spacing detected.<br>Pass `&:invoice_items` as an argument to `map` instead of a block.